script/bootstrap

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 set -e
+source script/check_go_version
```

Comment:
I think the whole external script approach might be a bit of an overkill. How about:

```sh
go_ver="$(go version | grep -o '1.[[:digit:]]\+')"
if [ "${go_ver#*.}" -lt 8 ]; then
  echo "You need to install Go 1.8 or higher to build hub" >&2
  STATUS=1
fi
```